_rules/html-has-title-2779a5.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: 2779a5
-name: HTML Page has a title
+name: HTML pages have a title
```

Comment:
Same comment re: pages vs page

Reply:
Using plural seems to have a majority preference in the group (it wasn't mine), so I'm sticking with that until I hear otherwise.